source/_posts/routing-in-react.md

```diff
@@ -0,0 +1,387 @@
+---
```

Comment:
This post should at least contain 2 presentational graphics. For example it can be an image, a chart, a presentation of some concepts, screenshot etc. You can add inside your post while you talk about some cool thing. You could create an image how something works or communicate with anything.

Reply:
We already agreed to add some visuals. @saurabhghatnekar I sent you some examples of graphics here: https://speakerdeck.com/pietrucha/angular-router-lets-get-lazy-1
![img](https://user-images.githubusercontent.com/8076187/224547534-2c16d879-3cb3-4793-95ad-2eda2da949bf.png)
- [ ] add visuals explaining how the router works

Reply:
There is only one image. We could add something else as well. Any infographic, screenshot. There is only a wall of text.